activesupport/test/cache/stores/mem_cache_store_test.rb

```diff
@@ -127,9 +127,17 @@ def test_local_cache_raw_values
     end
   end
 
+  def test_increment_initial
+    cache = lookup_store(raw: true)
+    cache.with_local_cache do
+      cache.increment("foo", 1, initial: 1)
```

Comment:
If the key doesn't exist, `increment` now sets it to the given value even when `initial` isn't passed, so we should add a test for that case and document it.

Passing `initial: nil` will restore the old behaviour of failing for a non-existent key; I think we should test and document that case too.